src/java/org/apache/cassandra/db/compaction/CompactionStrategyManager.java

```diff
@@ -141,6 +146,8 @@ should update the compaction strategy in maybeReload() due to an ALTER.
     private volatile long maxSSTableSizeBytes;
     private volatile String name;
 
+    public static int TWCS_BUCKET_COUNT_MAX = 30;
```

Comment:
I know we recommend 20-30 buckets, but if there are 31, are we just going to silently cut off the oldest one?

Reply:
Yes. We could up this, make it a configurable, both, neither, ? Nothing jumps out at me as an immediately convenient way to indicate to the end user that the value has been truncated when hitting up the JMX endpoint, and since we're not including it in the `tablestats` output we don't have that outlet to indicate truncation. You have any clever ideas here? :)

Reply:
The only easy improvement I can think of is to raise it to something that won't truncate w/ a slight misconfiguration. 128 maybe? Failing the JMX call because there are too many buckets seems like a bad option...

@krummas Thoughts?